README.rst

```diff
@@ -79,8 +115,18 @@ Finding and configuring device name in **NI Hardware Configuration Utility**:
   :align: center
   :width: 800px
 
-Tasks in NI-DAQmx
------------------
+Python Examples
+===============
+
+You can find a variety of examples in the GitHub repository in the
+`nidaqmx-python examples <https://github.com/ni/nidaqmx-python/tree/master/examples>`_
+directory.
+
+Core Concepts in NI-DAQmx
```

Comment:
Consider using "NI-DAQmx Key Concepts" to match https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/keyconcepts.html. Unless you think it will confuse people to call it the same thing? Most of these topics link there anyway.

Reply:
Nah, I like it.